src/main/java/ca/team2706/frc/robot/commands/lift/MoveLiftToPosition.java

```diff
@@ -43,6 +45,13 @@ protected boolean isFinished() {
 
     @Override
     protected void end() {
+        Log.d("Ended lift moving");

```

Comment:
I think it would be a good idea to log the final lift position here, just to see what kind of tolerances we're dealing with.